frontend/taipy-gui/base/src/app.ts

```diff
@@ -60,6 +62,16 @@ export class TaipyApp {
         this._onChange = handler;
     }
 
+    get onNotify() {
+        return this._onNotify;
+    }
+    set onNotify(handler: OnNotifyHandler | undefined) {
+        if (handler !== undefined && handler?.length !== 3) {
```

Comment:
I didn't you could get the number of parameters with function.length ...

Reply:
we could and it is very simple as you see!